### DISCUSSION TITLE
How do I set up a site for a domain and not a port?

### DISCUSSION THREAD

**Discussion by StarLabsUA** [requarks/wiki#3868] [Help / Questions]
Hello everyone, there is an important question.
How do I set up a site for a domain and not a port?
Now like this:
188.x.x.x:3000
It is necessary through the domain, I did not find instructions on the site (
Using debian 9

**Comment by Phil-hostwiki** [comment]
You can make use of port 80 instead of port 3000.
Are you using docker?

**Comment by mhasgari** [comment]
I suggest to utilize Traefik inside Docker Compose, in combination with WikiJS.
The results would be awesome to route from every domain or subdomain you want to your Wiki or even Wikies...
